### PR TITLE
don't add TLF to favorite in GetTLFCryptKeys

### PR DESCRIPTION
**Pull request by songgao**
It seems using a context key for this makes code easier to understand and maintain, than passing a parameter around.

### CODE REVIEW

File: libkbfs/keybase_service_base.go [hunk 1]
```diff
@@ -649,6 +649,8 @@ func (k *KeybaseServiceBase) GetTLFCryptKeys(ctx context.Context,
 		return keybase1.GetTLFCryptKeysRes{}, err
 	}
 
+	ctx = context.WithValue(ctx, CtxNoAddToFavoriteKey{}, true)
```
Comment (strib) [inline libkbfs/keybase_service_base.go]: `true` seems weird, since setting it to `false` doesn't do the opposite.  Maybe just set it to `struct{}{}`?

**Comment by songgao** [comment]
Hmm I think you convinced me using context key here is a bad idea -- It can be easily ignored. It's not really a lot of plumbing, but I did want to have a more formalized/unified way to specify whether an operation should be added to favorites.

**Comment by songgao** [comment]
@strib I added a `FavoritesOp` for this. PTAL Thanks!

**Comment by songgao** [comment]
@rjkmurray It's a ticket ID in our internal issue tracker

**Pull request merged**